React-Calculator/src/components/CalculatorShell.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import WolfButtonGrid from './WolfButtonGrid';
+import Screen from './Screen';
+
+export function CalculatorShell({ accumulator, isOff, addData }){
+    return (
```

Comment:
it may be a good idea to add the regular calculator to this abstraction. Should be fairly easy to do, but you'd probably have to modify addData also to contemplate for wolfram's `=` and the regular calc's `=`